### ISSUE TITLE
Title: Request for latest dataset_generation_coloradar.py and clarification on Coloradar calibration

### ISSUE THREAD

**Issue by evan20010126** [ZJU-FAST-Lab/Radar-Diffusion#16]
Hi, thanks for your great work!

I would like to ask whether it is possible to release the latest version of `dataset_generation_coloradar.py`. I am currently using the Coloradar dataset with the provided calibration parameters to perform LiDAR-to-Radar calibration. However, when I run inference on this pre-processed dataset using the released edm280k model weights, I am unable to reproduce the results reported in the paper.

Having access to your latest data-generation script would help me understand whether there are differences in preprocessing steps that might affect the reproduction of the results.

Thank you very much for your time, and I really appreciate any guidance or updates you can provide!

**Comment by RoboticsZhang** [comment]
Hello, there has been a server crash (about 1.5 years ago), all the generated dataset and checkpoints were lost.
The current released checkpoints were retrained on the newly generated dataset, and I just tested the EDM-280k on "aspen" scenario, obtaining the below result:
<img width="291" height="82" alt="Image" src="https://github.com/user-attachments/assets/7eba6d0c-54b1-44ca-ae95-bd1cb4961be6" />
The chamfer_distance and H_distance are indeed lower than the results we achieved with the original checkpoints. I am sorry for that and I guess there are 2 underlying reasons: 
1. The codebase for Radar-Diffusion is rather rudimentary and we haven't exert performance monitoring during training.  When we retrained the model after server crash, we just ran 280K and 140K steps training, the same as the original trial,  which cannot garantee the same performance. I suggest you split a validation set and monitor the performance during training , and select the best checkpoint.
2. In my memory， the key parameters for dataset generation are **remove_ceiling_flag** in lidar_preprocessing and **percentage** as well and **divide_num** in remove_ceiling(), because some scenes were recorded indoors, while others were outdoors. To filter out the ceiling cleanly, different scenes require different parameters. Regrettably, the parameters corresponding to each scenario have also been lost. As a result, the newly generated dataset we created differs from that in the paper, which might have also affected performance. I suggest you tune the parameters or try some other methods to clean out the ceiling.

If you still can't reproduce the results in the paper, then you can just use your own results, as long as a fair comparison is ensured, that is, training is done on the same dataset. You can also quote this page. 

Let me know if there is any progress.

**Comment by evan20010126** [comment]
Hi, thank you for the detailed explanation.
Based on your suggestions, I will regenerate the dataset using my preprocessing pipeline and train a new model from scratch. This way, all methods will be compared fairly on the same dataset version.

Thanks again for your help!
